src/resolvers/workspace.js

```diff
@@ -319,6 +320,61 @@ module.exports = {
       return true;
     },
 
+    /**
+     * Mutation in order to leave workspace
+     * @param {ResolverObj} _obj - object that contains the result returned from the resolver on the parent field
+     * @param {string} workspaceId - id of the workspace where the user should be removed
+     * @param {UserInContext} user - current authorized user {@see ../index.js}
+     * @param {ContextFactories} factories - factories for working with models
+     * @return {Promise<boolean>} - true if operation is successful
+     */
+    async deleteWorkspace(_obj, { workspaceId }, { user, factories }) {
+      const workspaceModel = await factories.workspacesFactory.findById(workspaceId);
+
+      if (!workspaceModel) {
+        throw new UserInputError('There is no workspace with provided id');
+      }
+
+      const memberInfo = await workspaceModel.getMemberInfo(user.id);
+
+      if (memberInfo.isAdmin) {
```

Comment:
big statement, should be inverted

```js
if (!memberInfo.isAdmin){
  ...
}

...
```

Reply:
Since only admin can delete the workspace So, why this should be inverted....?

Reply:
it is only about code style (removing extra indentation), not logic.

```js
if (statement1){
  if (statement2){
    // logic here
  } 
}
```

```js
if (!statement1){
  return; 
}

if (!statement2){
  return; 
}

// logic here
```